confluent_kafka/src/confluent_kafka.c

```diff
@@ -249,6 +249,39 @@ static PyObject* KafkaError_richcompare (KafkaError *self, PyObject *o2,
 	return result;
 }
 
+static PyObject *KafkaError_new (PyTypeObject *type, PyObject *args,
+                                 PyObject *kwargs) {
+        return type->tp_alloc(type, 0);
+}
+
+/**
+ * @brief Initialize a KafkaError object.
+ */
+static void KafkaError_init (KafkaError *self,
+			     rd_kafka_resp_err_t code, const char *str) {
+	self->code = code;
+        self->fatal = 0;
+        self->retriable = 0;
+        self->txn_requires_abort = 0;
+	if (str)
+		self->str = strdup(str);
+	else
+		self->str = NULL;
+}
+
+static int KafkaError_init0 (PyObject *selfobj, PyObject *args, PyObject *kwargs) {
+        KafkaError *self = (KafkaError *)selfobj;
+        int code;
+        const char *reason = NULL;
+ 
```

Comment:
remove empty line.

Reply:
sorry this came back when I added the additional args. Fixing again